### DISCUSSION TITLE
Why is this happening?

### DISCUSSION THREAD

**Discussion by Canonballss** [thepeacockproject/Peacock#540] [Q&A]
https://github.com/user-attachments/assets/f6faa92b-be7a-45d0-9fb2-6e6fc049f368

**Comment by Canonballss** [comment]
I just realised that the video doesnt work properly...

**Comment by AnthonyFuller** [comment]
I have no idea what you're trying to show, but you selected something in the server window which means it'll stop processing requests. You can tell as there's a section highlighted in white, next time press Escape to unselect it.
